From f4934e525c8852f779736c6857b4c73cac57fd0a Mon Sep 17 00:00:00 2001
From: "Kirill A. Korinsky" <kirill@korins.ky>
Date: Sat, 18 Sep 2021 02:18:43 +0200
Subject: [PATCH] Switch to using confuse

This commit fixed a warning:
```
beets/util/confit.py:21: UserWarning: beets.util.confit is deprecated; use confuse instead
  warnings.warn("beets.util.confit is deprecated; use confuse instead")
```
---
 beetsplug/ibroadcast/__init__.py | 2 +-
 setup.py                         | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/beetsplug/ibroadcast/__init__.py b/beetsplug/ibroadcast/__init__.py
index 2e8d436..52cd4a5 100644
--- a/beetsplug/ibroadcast/__init__.py
+++ b/beetsplug/ibroadcast/__init__.py
@@ -4,7 +4,7 @@
 import os

 from beets.plugins import BeetsPlugin
-from beets.util.confit import ConfigSource, load_yaml
+from confuse import ConfigSource, load_yaml

 from beetsplug.ibroadcast.command import IBroadcastCommand

diff --git a/setup.py b/setup.py
index 5bc3b12..b829807 100644
--- a/setup.py
+++ b/setup.py
@@ -38,6 +38,7 @@
     install_requires=[
         'beets>=1.4.9',
         'ibroadcast>=1.1.1',
+        'confuse',
     ],

     tests_require=[
